web_src/css/themes/theme-arc-green.css

```diff
@@ -124,19 +124,19 @@
   --color-orange-badge-hover-bg: #f2711c4d;
   --color-git: #f05133;
   /* target-based colors */
-  --color-body: #383c4a;
+  --color-body: #373b47;
   --color-box-header: #404652;
   --color-box-body: #2a2e3a;
   --color-box-body-highlight: #353945;
   --color-text-dark: #dbe0ea;
-  --color-text: #bbc0ca;
-  --color-text-light: #a6aab5;
-  --color-text-light-1: #979ba6;
-  --color-text-light-2: #8a8e99;
-  --color-text-light-3: #707687;
+  --color-text: #cbd0da;
+  --color-text-light: #bbbfca;
+  --color-text-light-1: #aaafb9;
+  --color-text-light-2: #9a9ea9;
+  --color-text-light-3: #8a8e99;
   --color-footer: #2e323e;
   --color-timeline: #4c525e;
-  --color-input-text: #d5dbe6;
+  --color-input-text: #dfe3ec;
   --color-input-background: #232933;
   --color-input-toggle-background: #454a57;
   --color-input-border: #454a57;
```

Comment:
I guess the input-border color is too dark, I can not find the radioboxes. 🤣 

![image](https://user-images.githubusercontent.com/2114189/229424560-e28714ee-9da8-4417-8ada-375198021b32.png)

Reply:
Yeah that needs improvement or convertion to native :)